firebase-app-distribution/src/test/java/com/google/firebase/appdistribution/FirebaseAppDistributionTest.java

```diff
@@ -233,9 +251,23 @@ public void updateAppTask_whenAabReleaseAvailable_redirectsToPlay() throws Excep
     assertEquals(UpdateStatus.REDIRECTED_TO_PLAY, taskResult.getUpdateStatus());
   }
 
+  @Test
+  public void updateApp_whenNotSignedIn_throwsError() throws Exception {
+    Mockito.doReturn(false).when(signInStorage).getSignInStatus();
+    firebaseAppDistribution.onActivityResumed(activity);
+
+    UpdateTask task = firebaseAppDistribution.updateApp();
+
+    assertTrue(task.getException() instanceof FirebaseAppDistributionException);
+    FirebaseAppDistributionException e = (FirebaseAppDistributionException) task.getException();
+    assertEquals(Constants.ErrorMessages.AUTHENTICATION_ERROR, e.getMessage());
+    assertEquals(AUTHENTICATION_FAILURE, e.getErrorCode());
+  }
+
   @Test
   public void updateToLatestRelease_whenNewAabReleaseAvailable_showsUpdateDialog()
       throws Exception {
+
```

Comment:
let's specify a return value for the istesterSignedIn method so it's clear what the expected signIn behavior is